quickstarts/flashdb/config.yml

```
@@ -0,0 +1,56 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: flashdb
+
+# Displayed in the UI (required)
+title: FlashDB
+
+# Long-form description of the quickstart (required)
+description: |
+  FlashDB is the fastest and lightest database in the universe. First, it can handle a large number of concurrent users without slowing down. Second, it has a very low latency, so queries are processed quickly. And finally, it scales easily as your data grows.
+  
+  This quickstart allows users to monitor their database to ensure peak performance. It also notifies you if there's any problem the database. 
+  
+  The flashdb quickstart includes: 
+  ## a dashboard 
+    where users can easily monitor:
+    - Database methods
+    - Average response time
+    - Errors
+    - Database size
+    - Cache hits
+    - Keys
+
+  ## alert conditions
+    where users are notified on:
+    - slow read responses
+    - low cache hit 
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  This quickstart allows users to monitor their flashdb database to ensure peak performance. It also notifies you if there's any problem the database. 
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Mehreen Tahir
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - infrastructure
+  - database
+
+alerts:
```

Hey @mehreentahir16! This is why they aren't showing up in the preview!
```suggestion
alertPolicies:
```